### PR TITLE
Extra branch

### PR DESCRIPTION
**Pull request by nithindante**
Please review the below code

### CODE REVIEW

File: .project
Comment (techcoachralph) [inline .project:1]: this should be in the .gitignore

File: .settings/org.eclipse.jdt.core.prefs
Comment (techcoachralph) [inline .settings/org.eclipse.jdt.core.prefs:1]: also in .gitignore

File: .settings/org.eclipse.m2e.core.prefs
Comment (techcoachralph) [inline .settings/org.eclipse.m2e.core.prefs:1]: also in .gitignore

File: src/main/java/AbstractComponents/AbstractComponents.java [hunk 1]
```diff
@@ -2,27 +2,27 @@
...
+import org.openqa.selenium.By; 
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
 import pageObjects.Basepackage;
 
-public class AbstractComponents extends Basepackage{
+public class AbstractComponents extends Basepackage{					//Creating a class to to avail all the common functions used within each classes.
```
Comment (techcoachralph) [inline src/main/java/AbstractComponents/AbstractComponents.java]: use the correct naming conventions and your class name is not descriptive

File: src/main/java/pageObjects/Basepackage.java
Comment (techcoachralph) [inline src/main/java/pageObjects/Basepackage.java:1]: your class name doesn't follow proper naming conventions

**Review by techcoachralph — CHANGES_REQUESTED** [review]
Follow the proper naming conventions as advised the other day and submit again for review. This seems like almost the exact same PR with minor changes.

File: src/main/java/commonMethods/CommonMethods.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package AbstractComponents;
+package commonMethods;
```
Comment (techcoachralph) [inline src/main/java/commonMethods/CommonMethods.java:1]: package names are supposed to be all in lower case `commonmethods` or you can have `common.methods`

File: src/main/java/commonMethods/CommonMethods.java [hunk 3]
```diff
@@ -8,17 +8,17 @@
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.openqa.selenium.support.ui.WebDriverWait;
 
-import pageObjects.Basepackage;
+import pageObjects.BasePackage;
```
Comment (techcoachralph) [inline src/main/java/commonMethods/CommonMethods.java:11]: package name again

File: src/main/java/pageObjects/Cart.java [hunk 4]
```diff
@@ -6,38 +6,38 @@
...
  List<WebElement> cartlist = driver.findElements(By.xpath("//div[@data-bundleitem=\"absent\"]"));
 return cartlist;
 } 
-public void selectItem(String productname) throws InterruptedException  			//finding the exact product to delete from the string using Streams() function 
+public void selectItem(String product) throws InterruptedException  			//finding the exact product to delete from the string using Streams() function 
 {		
-	WebElement item = getCartList().stream().filter(s->s.findElement(By.cssSelector("span[class='a-truncate-cut']")).getText().contains(productname)).findFirst().orElse(null);		//selecting the desired product
+	WebElement item = getCartList().stream().filter(s->s.findElement(By.cssSelector("span[class='a-truncate-cut']")).getText().contains(product)).findFirst().orElse(null);		//selecting the desired product
 	Thread.sleep(2000);
 	WebElement selectedElement = item.findElement(By.xpath("div[@class='sc-list-item-content']//div//div[@class='sc-item-content-group']/child::div//span[@data-feature-id=\"delete\"]//span//input"));	//clicking the delete button within the selected item
```
Comment (techcoachralph) [inline src/main/java/pageObjects/Cart.java:35]: xpath isn't bad but would look for one that is a little cleaner or a little more concise if possible

**Review by techcoachralph — APPROVED** [review]

File: src/main/java/commonMethods/CommonMethods.java (hunk 2 above)
Comment (nithindante) [reply src/main/java/commonMethods/CommonMethods.java:1]: ooh, I thought if you are using two words anywhere in Java, you need to use a capital letter to split it

File: src/main/java/commonMethods/CommonMethods.java (hunk 3 above)
Comment (nithindante) [reply src/main/java/commonMethods/CommonMethods.java:11]: Sue, noted

File: src/main/java/pageObjects/Cart.java (hunk 4 above)
Comment (nithindante) [reply src/main/java/pageObjects/Cart.java:35]: yeah, that xpath  was the only solution from my end,  I mean, how to select that delete button in each item, Sure will  try for a better one

**Review by nithindante — COMMENTED** [review]
Thanks for the review

**Pull request merged**